Directory.Build.props

```diff
@@ -0,0 +1,18 @@
+<Project>
+  <PropertyGroup>
+
+    <LangVersion>7.2</LangVersion>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <IsPackable>false</IsPackable>
+    
+  </PropertyGroup>
+  <ItemGroup>
+    <!-- PrivateAssets="All" is on these packages to ensure they're development dependencies
+         and aren't included as transitionary dependencies to package consumers.
+         For clarity, without PrivateAssets marked here, anyone consuming Microsoft.Identity.Client
+         would also be forced to install these dependencies.  PrivateAssets avoids this problem. -->
+    <PackageReference Include="AsyncUsageAnalyzers" Version="1.0.0-alpha003" PrivateAssets="All" />
```

Comment:
FYI, these analyzers were deprecated 3 years ago and replaced by the VS threading analyzers: https://github.com/Microsoft/vs-threading

Reply:
In addition, AsyncUsageAnalyzers is not present in the feed we are consuming while Microsoft.VisualStudio.Threading is.